Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -90,17 +89,106 @@ if(WIN32)
         win32/res/Dirt.ICO
         win32/res/headspin.ico
     )
+elseif(APPLE)
+    include_directories("../../PubUtilLib/plPipeline/GL/OSX")
```

Comment:
This can be removed (these files don't exist at that path in any current branch)